database/factories/ApplicationFactory.php

```diff
@@ -88,4 +97,27 @@
             'skillable_type' => $skillableType,
         ]));
     }
+
+    $submitted = ApplicationStatus::where('name', 'submitted')->firstOrFail()->id;
+    if ($application->version_id === 2 && $application->application_status_id === $submitted) {
+        $criterion = $application->job_poster->criteria ->first();
+        $work = factory(ExperienceWork::class)->create([
+            'experienceable_id' => $application->applicant_id,
+            'experienceable_type' => 'application',
+        ]);
+        factory(ExperienceSkill::class)->create([
+            'skill_id' => $criterion->skill_id,
+            'experience_type' => 'experience_work',
+            'experience_id' => $work->id,
+        ]);
+        $education = factory(ExperienceEducation::class)->create([
+            'experienceable_id' => $application->id,
+            'experienceable_type' => 'application',
+        ]);
+        factory(ExperienceSkill::class)->create([
+            'skill_id' => $criterion->skill_id,
+            'experience_type' => 'experience_work',
+            'experience_id' => $education->id,
+        ]);
+    }
```

Comment:
This works, I might rework it a bit so it works for draft posters as well.

We may also want to make version 2 the default, but we might want to wait until the new application is live.

Reply:
Ya I wasn't sure what the best way to set this up. I opted setting the priority user to have a version 2 application, which is in `DevSeeder.php`.

Reply:
Good idea to make them priority users, it makes them easier to find.